test/e2e/dns_ingressdegrade_test.go

```diff
@@ -28,17 +26,12 @@ import (
 func TestIngressStatus(t *testing.T) {
 	kubeConfig, err := config.GetConfig()
 	if err != nil {
-		fmt.Printf("failed to get kube config: %s\n", err)
-		os.Exit(1)
+		t.Fatalf("failed to get kube config: %s\n", err)
 	}
 	kubeClient, err := operatorclient.NewClient(kubeConfig)
```

Comment:
We need to check for an error here. Looks like the previous check went missing in the update.

Reply:
Ooops my bad will sort that out now

Reply:
Let's squash some of the commits and we can tag.